openmdao/drivers/latinhypercube_driver.py

```diff
@@ -109,10 +109,14 @@ def mmphi(self):
             # TODO: This norm takes up the majority of the computation time. It
```

Comment:
I don't know. It seems really fast now and isn't really the high peg in the calculation, plus shouldn't scale geometrically anymore, so maybe not.